.github/workflows/test-pr.yml

```diff
@@ -11,9 +11,16 @@ jobs:
     - name: Install Poetry
       uses: snok/install-poetry@v1
     - run: poetry install
-    - run: poetry run coverage run -m pytest
+    - run: poetry run coverage run --branch -m pytest
+    - run: poetry run coverage json
     - name: Archive code coverage results
       uses: actions/upload-artifact@v3
       with:
         name: code-coverage-report
```

Comment:
Delete the entire step, not just the file path. There's no need to upload any artifact.